src/extension.ts

```diff
@@ -32,6 +35,12 @@ export async function activate(context: vscode.ExtensionContext) {
 	// Register the command, _after_ the list of targets has been read. This
 	// makes sure the user will never see an empty list.
 	let disposable = vscode.commands.registerCommand('vscode-tinygo.selectTarget', async () => {
+		let buildPkg = '';
+		let editor = vscode.window.activeTextEditor;
+		if (editor) {
+			buildPkg = editor.document.fileName;
+		}
```

Comment:
Go packages normally work in packages, not in files. I think this should use the directory of the current file, not the path of the current file.